integrationExamples/gpt/x-domain/creative.js

```diff
@@ -0,0 +1,53 @@
+// this script can be returned by an ad server delivering a cross domain iframe, into which the
+// creative will be rendered, e.g. DFP delivering a SafeFrame
+
+const publisherDomain = 'http://localhost:9999';
+const adServerDomain = 'http://tpc.googlesyndication.com';
```

Comment:
Let's add a note about making sure to match the protocol. I assume that this becomes HTTPS on secure pages?